handlers/services.go

```diff
@@ -60,3 +60,16 @@ func ServiceMetrics(w http.ResponseWriter, r *http.Request) {
 	}
 	RespondWithJSON(w, http.StatusOK, metrics)
 }
+
+func ServiceDetails(w http.ResponseWriter, r *http.Request) {
+	params := mux.Vars(r)
+
+	serviceDetails, err := models.GetServiceDetails(params["namespace"], params["service"])
+	if err != nil {
```

Comment:
It would be nice to make a distinction between real errors and 404/not found.
Not a blocker though.